packages/vue/src/components/organisms/SfFooter/SfFooter.vue

```diff
@@ -62,6 +73,17 @@ export default {
       }
       this.$emit("change", this.open);
     },
+    /* 
+      function to decode string into HTML
+    */
+    decodeString() {
```

Comment:
@AdamPawlinski can you explain this method? :)

Reply:
It is used to properly add special sign like & in "Payment & Delivery"

Reply:
@AdamPawlinski I checked it directly and it works without this hack ;) You can't open "Payment & Delivery" from storybook because it does something with data before use it :) 
![Zrzut ekranu 2020-06-5 o 11 31 32](https://user-images.githubusercontent.com/12138170/83861323-ba9f1000-a720-11ea-855d-434f8622aeb0.png)

Reply:
Thanks for the review :)